cmd/configschema/go.mod

```diff
@@ -5,11 +5,171 @@ go 1.18
 require (
 	github.com/fatih/structtag v1.2.0
 	github.com/google/uuid v1.3.0
-	github.com/open-telemetry/opentelemetry-collector-contrib v0.70.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/alibabacloudlogserviceexporter v0.70.0
```

Comment:
These versions are using `v0.70.0`, it should be `v0.73.0` right?

Reply:
this PR is really old now, see #19456 for a new take